.pre-commit-config.yaml

```diff
@@ -30,6 +30,8 @@
         aiida/backends/djsite/db/migrations/0015_invalidating_node_hash.py|
         aiida/backends/djsite/db/migrations/0016_code_sub_class_of_data.py|
         aiida/backends/djsite/db/migrations/0017_drop_dbcalcstate.py|
+        aiida/backends/djsite/db/migrations/0021_dbgroup_name2label_type2type_string.py|
+        aiida/backends/djsite/db/migrations/0022_dbgroup_type_string_change_content.py|
```

Comment:
Please remove the migrations from the `pre-commit.yaml`

Reply:
done